### DISCUSSION TITLE
Items moving in player inventory

### DISCUSSION THREAD

**Discussion by SnowmanFTW** [Minestom/Minestom#1897] [Help]
Did i do something wrong to cancel/break the ability to move items in my inventory or does it need to actually be implement by me?
I already tried to run the server without any events other than login (i think, lol) so yeah im asking this since ive been pretty stuck on this for no reason

**Comment by ZakShearman** [comment]
Are you saying they're getting cancelled or that they're not getting cancelled?

You can move items in your inventory by default.

**Reply by SnowmanFTW** [reply]
I've put an isCancelled check in a preinventoryclickevent and its outputting as false but the item is for sure not going anywhere

Edit: just reread what you asked, the moving of the item is getting cancelled, tho im pretty sure i dont have anything handling inventory clicking, and as i said above, if i output the iscancelled method, it says false.
